docker/Dockerfile.cpu

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:16.04
+
+# MAINTAINER sshuair<sshuair@gmail.com>
+
+# install dependencies
+RUN apt-get update --fix-missing && apt-get install -y --no-install-recommends\ 
+        build-essential \
+        software-properties-common \
+        curl \
+        cmake \
+        libfreetype6-dev \
+        libpng12-dev \
+        libzmq3-dev \
+        libspatialindex-dev \
+        pkg-config \
+        rsync \
+        zip \
+        unzip \
+        git \
+        wget \
+        vim \
+        ca-certificates \
+        python3 \
+        python3-dev \
+        python3-pip \
+        ipython3 \
+        libboost-python-dev \
+        libexpat1-dev \
+        zlib1g-dev \
+        libbz2-dev \
+        libsm6 \
+        libxext6 \
+        libxrender1 \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# install python package
+RUN pip3 --no-cache-dir install setuptools && \
+    pip3 --no-cache-dir install http://download.pytorch.org/whl/cpu/torch-0.4.0-cp35-cp35m-linux_x86_64.whl && \
+    pip3 --no-cache-dir install \
+        ipykernel \
+        jupyter \
+        torchvision \
```

Comment:
torchvision should be included in the `deps/requirements-lock.txt` below, no?

Reply:
ok, I will fix this